src/views/NodeDetail.vue

```diff
@@ -391,27 +391,34 @@ export default Vue.extend({
   methods: {
     async update() {
       this.loading = true;
-      const attributes = await api.attributes(this.workspace, this.graph, `${this.type}/${this.node}`);
-      const incoming = await api.edges(this.workspace, this.graph, `${this.type}/${this.node}`, {
+      const nodeTable = (await api.table(this.workspace, this.type, {})).results;
+      // eslint-disable-next-line no-underscore-dangle
+      const node = nodeTable[nodeTable.map((table) => table._key).indexOf(this.node)];
```

Comment:
I'm almost certain you can use something like [`Array.find()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find) here instead of this rather complex array indexing expression.

Reply:
See https://github.com/multinet-app/multinet-client/commit/ceedc898a945927e4e277d13583c9eec195e6c64#diff-24af31592a6444e4718ffd1207989976b07faaae1e61bc74c9da27b0537adbf9L396-R396